src/components/UI/Dropdown/index.js

```diff
@@ -45,13 +60,14 @@ export function StyledDropdown(props) {
     }
   }, [props.list]);
 
+  //Set input width acording to filter key word
   useEffect(() => {
     var length = '300px';
-    if (filter.length > 0) length = filter.length;
+    if (filterWord.length > 0) length = filterWord.length;
```

Comment:
This will only work if you're using a monospace font (where all characters are same width).
So, you have to make sure that fallback fonts are also monospace, or the word could exceed the width of the input.

Reply:
Thanks for the heads up, I didn't know that was a term.

Reply:
https://meyerweb.com/eric/thoughts/2018/06/28/what-is-the-css-ch-unit/